examples/psr7.php

```diff
@@ -50,6 +51,13 @@ public function json()
 
         return print_r($payload, true);
     }
+
+    public function url()
```

Comment:
Maybe we can keep out the examples :) They're intended to show just the basic operations